modules/olm-installing-opm.adoc

```diff
@@ -16,7 +16,9 @@ You can install the `opm` CLI tool on your Linux, macOS, or Windows workstation.
 
 .Prerequisites
 
-* `podman` version 1.4.4+
+* If you use a Linux as your workstation operating system, it must use the following packages:
```

Comment:
```suggestion
* Linux dependencies:
```

```

Reply:
We shouldn't assume `opm` is being installed only to the desktop or workstation -- users can install it in pipeline scenarios to build/validate bundles and in containers to build custom indexes

Reply:
@kalexand-rh how about your thought on this suggestion ?

Reply:
I'm fine with removing "workstation." It's a valid concern that I had not considered.

Reply:
@kalexand-rh Thank you for your confirmation. @exdx Thank you for your suggestion. I've modified again as you mentioned.